tests/Umbraco.Tests.Integration/ManagementApi/Services/Trees/FileSystemTreeServiceTestsBase.cs

```diff
@@ -41,7 +41,7 @@ public void SetUpFileSystem()
         }
     }
 
-    private static Stream CreateStream(string contents = null)
+    public Stream CreateStream(string contents = null)
```

Comment:
Can be `protected` I believe, which improves encapsulation (even if it is a test class, still good practice).